riff-raff/app/AppComponents.scala

```diff
@@ -148,8 +148,9 @@ class AppComponents(context: Context, config: Config, passwordProvider: Password
   val hooksClient = new HooksClient(datastore, hookConfigRepository, wsClient, executionContext)
 
   val shutdownWhenInactive = new ShutdownWhenInactive(deployments)
+  val rotateInstanceWhenInactive = new RotateInstanceWhenInactive(deployments, config)
```

Comment:
A real nitpick, but we're not performing a rotation directly here - that's a side effect performed by the ASG. Not important but you could go with `ShutdownAppWhenInactive` and `TerminateInstanceWhenInactive` as an alternative.